code/datums/components/aquarium.dm

```diff
@@ -16,6 +16,11 @@
 	//Current layer for the visual object
 	var/base_layer
 
+	/// Is the flopping animation allowed. Animation was made for fish sprites so will look off on other things so consider toggling this.
+	var/floppable = TRUE
```

Comment:
If the animation was made for fish sprites, why not move the animation origin to the /obj/item/fish, item? To be honest, I'm still not 100% on the reasoning for the weird not-quite-an-item-datum here, so I'll defer if you think it's a good idea.

Just `floppable`, `flopping`. We're only a couple more vars `flopped`, `has_flopped`, `will_flop`.

Reply:
I was considering that, but this let's you admin abuse into flopping spacemen. I should probably add a component parameter for this though since right now it requires direct modification.